samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -15,7 +15,7 @@
 #include "storage.h"
 
 static struct bt_mesh_cfg_srv cfg_srv = {
-	.relay = BT_MESH_RELAY_DISABLED,
+	.relay = BT_MESH_RELAY_ENABLED,
```

Comment:
Wouldn't it make more sense to do this on a per-node basis? Ideally you shouldn't have multiple relays very close to each other since it's just redundant and will congest the available bandwidth.

Reply:
> .beacon = BT_MESH_BEACON_**ENABLED,**
> 
> /* 5 transmissions with 30ms interval */
> .net_transmit = BT_MESH_TRANSMIT(5, **30**),
> 
> /* 5 transmissions with 20ms interval */
> .relay_retransmit = BT_MESH_TRANSMIT(5, **20**),

As of now #nRFMesh don't support Relay On/Off feature & that is why I enable it by default.

Based on my observation after these modification in 3 lines, I saw considerable improvement in Mesh network  (GATT Bearer) performance. All nodes are more behaved in sync manner plus all NODEs are accepting commands...means no packet loss !!

Previously, Android-nRFMesh app missed the GET status response or status message broadcast by NODE after receiving reliable message & hangs forever. But now this frequency of happening is very low. This may be Android App level issue.

I know previously, when I enabled Relay on every NODE that hangs mesh network considerably.  But now things have changed. I don't know why but recent modification are most satisfactory as of now. Plus I don't need to add following lines in .... subsys/bluetooth/host/mesh/adv.c ....adv_thread()...

>                         u8_t delay;
> 
>                         BT_MESH_ADV(buf)->busy = 0;
> 
>                        /* Add 0-50ms random delay to transmissions in
>                         * order to reduce collisions.
>                         */
>                        bt_rand(&delay, sizeof(delay));
>                        k_sleep(K_MSEC(delay % 50));
> 
>                        adv_send(buf);